src/antsibull/data/ansible-email-announcement.j2

```diff
@@ -40,6 +40,8 @@ SHA256:
 
 {{ "Some important details" | email_heading }}
 
+ansible-core is a required dependency, not contained within the ansible package. Pip install installs the dependency, but it can also be installed independently of the ansible package.
```

Comment:
```suggestion
ansible-core is a separate package on which ansible depends. `pip install ansible` installs `ansible-core`, but it can also be installed independently of the ansible package.
```

Reply:
80edeb6568194868713fe301528c7d2ae8b6bcd5